cpp/betweenness_centrality_module/CMakeLists.txt

```diff
@@ -4,6 +4,13 @@ set(betweenness_centrality_src
 
 add_query_module(betweenness_centrality 1 "${betweenness_centrality_src}")
 
+find_package(OpenMP)
+if (OPENMP_FOUND)
+    set (CMAKE_C_FLAGS "${CMAKE_C_FLAGS} ${OpenMP_C_FLAGS}")
+    set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${OpenMP_CXX_FLAGS}")
+    set (CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} ${OpenMP_EXE_LINKER_FLAGS}")
+endif()
```

Comment:
@antonio2368 Should we use `AddExternalProject` to manage OpenMP dependency?

Reply:
yes! It supports sending all those specified arguments above.